### PR TITLE
github test

### PR DESCRIPTION
**Pull request by OrkunTokdemir**
# Qt contribution guidelines

We welcome contributions to Qt!

Read the
[Qt Contribution Guidelines](https://wiki.qt.io/Qt_Contribution_Guidelines) to learn more.

### CODE REVIEW

**Comment by OrkunTokdemir** [comment]
Sas

**Review by benchoq — APPROVED** [review]
github testing: PR approval

**Comment by OrkunTokdemir** [comment]
Test discussion

**Review by OrkunTokdemir — COMMENTED** [review]
test discussion

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -4,3 +4,5 @@ We welcome contributions to Qt!
 
 Read the
 [Qt Contribution Guidelines](https://wiki.qt.io/Qt_Contribution_Guidelines) to learn more.
+
+github test
```
Comment (OrkunTokdemir) [inline .github/pull_request_template.md:8]: test discussion

Comment (OrkunTokdemir) [inline .github/pull_request_template.md:8]: test discussion

**Pull request closed without merging**